spec/imsc-hrm.html

```diff
@@ -222,6 +222,9 @@ <h2>Terms and Definitions</h2>
       
       <p><dfn>empty ISD</dfn>. an <a>Intermediate Synchronic Document</a> with no <dfn
       data-cite="IMSC#dfn-presented-region">presented region</dfn>.</p>
+
+      <p class="note">Where terms used are not otherwise defined, and are glossed in [[i18n-glossary]],
```

Comment:
This is too vague and should be removed.

Reply:
Yes, I would use a `<dfn data-cite="url">term</dfn>` when it is first use and `<a>term</a>` elsewhere. We could also say _term, as defined at [[XXXX]]_.

Reply:
I'm not delighted by the way that Respec handles definitions that are themselves citations: whereas with local terms without `data-cite` a click reveals all the references to the term in the document, one with a `data-cite` takes you to the original definition. So choosing the latter option.